lunes_cms/static/css/feedback.css

```diff
@@ -8,34 +8,48 @@ table {
   min-width: 1000px;
 }
 
+
+.table-striped tbody tr:nth-of-type(odd){
+  background-color: #ffffff;
```

Comment:
I think it could make sense to [introduce CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) so that we can reuse them. But I don't want to block this PR any longer and therefore I suggest that we could open up a short clean-up PR afterwards.

Reply:
We probably have to adjust the design of other tables, for some header text in them are gray outed too 🤔 When all the important issues are done, I'll come back to CSS/js refactoring.